### ISSUE TITLE
WCTE validation

### ISSUE THREAD

**Issue by tdealtry** [WCSim/Validation#17]
Now that there's a WCTE geometry in WCSim/WCSim, it'd be nice to automate WCTE validation, in the same way we are validating SK/HK FD(+OD)/IWCD(+OD)

I've added some mac files to do this, using the typical values used for other detectors
- 5 MeV electron @ detector centre
- 500 MeV electron @ detector centre
- 500 MeV muon @ detector centre

However the high-energy ones are not playing nicely. It seems that they are killing the runner, when trying to `cat` to screen the WCSim job output. Why is this happening? It seems to be a glut of GeomNav1002 warnings (check the .txt files [here](https://github.com/WCSim/Validation/compare/new_ref?expand=1))

A trivial workaround would be to cut down the number of events used in each file to say 100. But @kmtsui can you advise on some more useful tests? e.g. What are the expected particles/energies WCTE is expected to see in the test beam?

[Ultimately it'd be nice to fix the masses of GeomNav1002 warnings we see in WCTE & IWCD, but that is highly non-trivial]

**Comment by kmtsui** [comment]
Yes I am aware of the GeomNav1002 warnings. I suspect this is due to the much smaller detector size which causes more overlapping in geometry.
One possible fix is to use the non-replica detector construction method (Declare `/WCSim/PMT/ReplicaPlacement false` before `/WCSim/Construct`). This will remove most of the GeomNav1002 warnings, but causes a small change in geofile output due to numerical accuracy.

**Comment by tdealtry** [comment]
That's fixed it. Thanks!
